config/config.go

```diff
@@ -869,14 +881,14 @@ func getTxFeeConfig(v *viper.Viper, networkID uint32) genesis.TxFeeConfig {
 
 func getGenesisData(v *viper.Viper, networkID uint32, stakingCfg *genesis.StakingConfig) ([]byte, ids.ID, error) {
 	// try first loading genesis content directly from flag/env-var
-	if v.IsSet(GenesisConfigContentKey) {
-		genesisData := v.GetString(GenesisConfigContentKey)
+	if v.IsSet(getRenamedKey(v, GenesisConfigContentKey, GenesisFileContentKey)) {
```

Comment:
Nit: can we define a var for `getRenamedKey(v, GenesisConfigContentKey, GenesisFileContentKey)` and use that twice instead of calling the function twice?

Same for `getRenamedKey(v, GenesisConfigFileKey, GenesisFileKey)` below

Reply:
changed